package.json

```diff
@@ -111,6 +111,7 @@
   "scripts": {
     "lint": "jshint . && node bin/jscs lib test bin publish",
     "test": "npm run lint && mocha",
+    "test-appveyor": "mocha --color",
```

Comment:
Copy https://github.com/jscs-dev/node-jscs/pull/937#issuecomment-72307626.

`supports-color` code has this [`isTTY` condition](https://github.com/sindresorhus/supports-color/blob/2ae39320b7ca26c2803b4da531ed9bbcc40d78a3/index.js#L18), which satisfies for AppVeyor as teletyping is not supported in AppVeyor. On Travis CI, it does not satisfies this condition as stdout has `isTTY` set to `true`.

I made this [test commit](https://github.com/am11/node-jscs/commit/bc518d41e1729f02aa4a30aa0ea27383d52f771c) to verify it. [appveyor result](https://ci.appveyor.com/project/am11/node-jscs/build/33#L148) and here is [travis-ci result](https://travis-ci.org/jscs-dev/node-jscs/builds/49057659#L94). I will rebase it out now.